image_proc/src/debayer.cpp

```diff
@@ -56,10 +56,14 @@ DebayerNode::DebayerNode(const rclcpp::NodeOptions & options)
   // TransportHints does not actually declare the parameter
   this->declare_parameter<std::string>("image_transport", "raw");
 
+  debayer_ = this->declare_parameter("debayer", 3);
```

Comment:
slightly unrelated fix - but I noticed this parameter gets declared after we create publisher (and publisher callback) - so I moved this up with other parameters